src/api/FileActivities.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * @flow
+ * @file Helper for the box File Activity API
+ * @author Box
+ */
+
+import Base from './Base';
+import { PERMISSION_CAN_COMMENT, PERMISSION_CAN_VIEW_ANNOTATIONS, ERROR_CODE_FETCH_ACTIVITY } from '../constants';
+import type { BoxItemPermission } from '../common/types/core';
+import type { ElementsXhrError } from '../common/types/api';
+import type { FileActivity, FileActivityTypes } from '../common/types/feed';
+
+const getFileActivityQueryParams = (
+    fileID: string,
+    activityTypes?: FileActivityTypes[] = [],
+    enableReplies?: boolean = true,
+) => {
+    const baseEndpoint = `/file_activities?file_id=${fileID}`;
+    const hasActivityTypes = !!activityTypes && !!activityTypes.length;
+    const enabledRepliesQueryParam = '&enable_replies=true&reply_limit=1';
+
+    return `${baseEndpoint}${hasActivityTypes ? `&activity_types=${activityTypes.join()}` : ''}${
```

Comment:
Still a bit tough to see what's going on here. Can we use the `query-string` library to help us? It's already in the dependencies list.

Reply:
Tried out using query-string, but it seems like our version is very limited in functionality. Seems like it cannot use commas as separators for arrays, as well as recognize commas in `stringify` so I couldn't use `join()` alongside it. To make the url easier to read, I put all of the ternaries in variables so the url consists of the baseUrl and queryParam variables.